package.json

```diff
@@ -49,23 +49,23 @@
     "bin/*": "standard"
   },
   "dependencies": {
+    "@babel/core": "7.0.0-beta.32",
+    "@babel/generator": "7.0.0-beta.32",
+    "@babel/plugin-proposal-class-properties": "7.0.0-beta.32",
+    "@babel/plugin-proposal-object-rest-spread": "7.0.0-beta.32",
+    "@babel/plugin-syntax-dynamic-import": "7.0.0-beta.32",
+    "@babel/plugin-transform-modules-commonjs": "7.0.0-beta.32",
+    "@babel/plugin-transform-react-jsx-source": "7.0.0-beta.32",
+    "@babel/plugin-transform-runtime": "7.0.0-beta.32",
+    "@babel/preset-env": "7.0.0-beta.32",
+    "@babel/preset-react": "7.0.0-beta.32",
+    "@babel/runtime": "7.0.0-beta.32",
+    "@babel/template": "7.0.0-beta.32",
     "ansi-html": "0.0.7",
-    "babel-core": "6.26.0",
```

Comment:
Looks like [this](https://github.com/zeit/next.js/blob/canary/server/build/babel/find-config.js#L2) is still using `babel-core`.

Reply:
You will need to change it `@babel/core`, but in general you should rely on Babel's internals. 

We already had a few discussions to expose the config to plugins authors https://github.com/babel/babel/pull/7259.

Reply:
If that PR gets merged we'll change to it.